daemon/containerd/image_delete.go

```diff
@@ -30,8 +38,6 @@ import (
 // are divided into two categories grouped by their severity:
 //
 // Hard Conflict:
-//   - a pull or build using the image.
```

Comment:
Mostly unrelated, but I did not find any check for this in the old implementation either 😅  We should probably remove it, unless I missed it somewhere (not even sure how we would check for this, or how a pull would be using the image – for build, I guess with the classic builder this would fall under the "check if a running container is using it"?).
cc @thaJeztah